src/App.css

```
@@ -1,3 +1,12 @@
-.center-text {
+.App {
   text-align: center;
-}
\ No newline at end of file
+}
+label {
+  display: block;
+  text-align: center;
+}
+
+.attendantsDisplay span {
```

Same as above. One concrete class would be enough e.g.
```suggestion
.attendants-title {
```